proto/internal/temporal/server/api/schedule/v1/message.proto

```diff
@@ -155,3 +159,55 @@ message HsmSchedulerState {
     google.protobuf.Timestamp next_invocation_time = 3;
 
 }
+
+// State machine scheduler internal state.
+message SchedulerInternal {
```

Comment:
nit: you probably don't want to type `Internal` every time in the Go code, it should be implied IMHO. Up to you.